### ISSUE TITLE
Create PR to torchvision for MovingMNIST

### ISSUE THREAD

**Issue by tsugumi-sys** [tsugumi-sys/SA-ConvLSTM-Pytorch#3]
https://github.com/pytorch/vision/issues/3562

**Comment by tsugumi-sys** [comment]
1. Transform for each frames: Done
2. There two ways (1: Read form file and split. 2: Generate dataset with this class and split.) I prefer 1 to 2 and[ tensorflow also do 1](https://github.com/tensorflow/datasets/blob/master/tensorflow_datasets/video/moving_mnist.py). [The moving sequence dataset implemented by tensorflow](https://github.com/tensorflow/datasets/blob/b9deaa07f567ad3211850a1e5c1ae993a1a03409/tensorflow_datasets/video/moving_sequence.py) is an other dataset.
3. Simply return a flames, Not split into input and label. : remove `input_seq_length`. Moving MNIST is used both supervised and unsupervised training,  so we should just return frames here.

**Comment by tsugumi-sys** [comment]
Add tests

- https://github.com/pytorch/vision/blob/main/test/test_datasets_download.py
- https://github.com/pytorch/vision/blob/main/test/test_datasets.py

Change docs

- https://github.com/pytorch/vision/blob/main/docs/source/datasets.rst
